src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,40 +1,41 @@
 package edu.kis.vh.nursery;
 
+import stack.IntLinkedList;
+
 public class DefaultCountingOutRhymer {
 
-	IntArrayStack intArrayStack= new IntArrayStack();
+	private final IntLinkedList intLinkedList;

```

Comment:
The goal of the lab was to make using IntLinkedList and IntArrayStack interchangeable.